core/src/main/java/org/elasticsearch/action/admin/cluster/allocation/NodeExplanation.java

```diff
@@ -108,7 +108,9 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
             builder.field("final_decision", finalDecision.toString());
```

Comment:
This is the old explain API code.  Once we plug the new explain API into the rest layer, this whole class will disappear.